src/css/flags/time-slider-above.less

```diff
@@ -187,8 +187,16 @@ of the file to be defined separately.
     }
 
     .jw-group {
-      .jw-icon-cast button {
-        width: 37px;
+      .jw-icon-cast {
+        width: auto;
+
+        button {
+          font-size: inherit;
+          width: 44px;
```

Comment:
It'd more consistent if we define width and height relative to @mobile-touch-target so that everything's pulling 44px from that variable.